app/etl/item/content/parsers/contact.rb

```diff
@@ -2,9 +2,12 @@ class Item::Content::Parsers::Contact
   def parse(json)
     html = []
     html << json.dig('details', 'description')
-    html << get_email(json)
-    html << get_postal(json)
-    html << get_phone(json)
+    emails = get_email(json)
+    html << emails unless emails.nil?
+    postals = get_postal(json)
+    html << postals unless postals.nil?
+    phones = get_phone(json)
+    html << phones unless phones.nil?
```

Comment:
[minor] I think you could use `html << get_email(json) unless get_email(json).nil?` to keep the method a bit shorter and easier to read.